dev-client/src/translations/en.json

```diff
@@ -21,17 +21,15 @@
         "link_url": "https://landpks.terraso.org/",
         "get_started": "Get started"
     },
-    "home": {
-        "info": {
-            "title": "Discover your land’s potential",
-            "description": "<0>Discover the potential use and value of your land.</0> Accurately determine the soil type and other features of your land. Gain insights for grazing and farming options, carbon sequestration, and more.\n\n<2>There are three ways to use the map:</2>",
-            "list1": "Tap the map to drop a pin and get more information about that location.",
-            "list2": "Tap the location icon <icon/> to find your current GPS location.",
-            "list3": "Enter coordinates, address, or search term in the search bar.",
-            "description2": "<bold>Find more information and guides</bold> at:\n<icon />",
-            "link_label": "landpotential.org",
-            "link_url": "https://landpotential.org/"
-        }
+    "info": {
```

Comment:
Requesting input on these changed translation keys -- is there a preferred way to update one (json, or po) and auto-update the other? Will renamed keys have consequences for our translation partner?

Reply:
I don't have the actual answer for this, but I have a little info on the general translation process from Paul in a [past comment](https://github.com/techmatters/terraso-mobile-client/pull/2166#discussion_r1770631315):
> en.json
> -> en.po (conversion script)
> -> POEditor -> translator fills in translations
> -> es.po (POEditor export)
> -> es.json (conversion script)

So I image we can generate the en.po from the en.json by running `npm run localization-to-po`, but I'm not sure what steps we should do for the Spanish files from there. Interested to know, when Paul gets back.

Reply:
I recommend updating JSON (both en and es) and generating the .po files with the script.

If POEditor doesn't resolve these cleanly, we can manually fix it.